dash_bootstrap_components/table.py

```diff
@@ -0,0 +1,85 @@
+import dash_bootstrap_components as dbc
+import dash_html_components as html
```

Comment:
Does this introduce a depencency on `dash_html_components`?

Reply:
Yes it does, good point. Should we put it in the dependencies?

Reply:
yes, I think we should (it's not like pandas because it's more lightweight and users of dbc are likely to need dash_html_components anyway, whereas they might not need pandas)